app/controllers/util/MultipleItemsHelper.scala

```diff
@@ -67,7 +67,7 @@ object MultipleItemsHelper {
     */
   def remove[A](idOpt: Option[String], cachedData: Seq[A]): Seq[A] = idOpt match {
     case Some(id) if validateId(id) && cachedData.length - 1 >= id.toInt => removeItem(id, cachedData)
-    case _                                                               => throw new InternalServerException("Incorrect id")
```

Comment:
🥇  I noticed we've stopped using this helper in a lot of the controllers, does it not work how we want it to?

Reply:
I've replaced this helper method.  We don't want to be removing by index.